controllers/src/main/scala/cromwell/pipeline/controller/ProjectFileController.scala

```diff
@@ -24,6 +24,17 @@ class ProjectFileController(wdlService: ProjectFileService)(implicit val executi
             }
           }
         }
+      },
+      path("files") {
+        post {
+          entity(as[ProjectUpdateFileRequest]) { request =>
+            onComplete(wdlService.uploadFile(request.project, request.projectFile)) {
+              case Success(Left(e)) => complete(StatusCodes.InternalServerError, e.getMessage)
```

Comment:
What exactly does it mean to get ```Left``` here? Is it reasonable to return server error? In case of invalid file code 4xx could be better

Reply:
Yep, my bad. But I don't know what invalid file code I should return exactly because I get only VersioningException with error message

Reply:
Fixed on `IamTeapot`

Reply:
Please add TODO. This status code should be fixed later, when `updareFile` will return good issues.

Reply:
Done